functests/xml-tests.yaml

```
@@ -87,6 +134,33 @@ tests:
       </unit>
     normalize: none
 
+  - name: "Balanced Ternary"
+    command: "{command} --format xml  --decimal"
+    input: |
+      -0t1TT01
+    expected_output: |
+      <unit>
+        <number decimal="-46" value="-0t1TT01" />
+      </unit>
+
+  - name: "Balanced Ternary"
+    command: "{command} --format xml  --decimal"
+    input: |
+      +0t1TT01
+    expected_output: |
+      <unit>
+        <number decimal="46" value="+0t1TT01" />
+      </unit>
+
+  - name: "Balanced Ternary"
```

This appears to be a duplicate test name, which could cause confusion if it was to fail. Could it be renamed?

```suggestion
  - name: "Unsigned Balanced Ternary"
```